docs/src/pages/community.tsx

```diff
@@ -20,103 +20,193 @@ import React from 'react';
+  }
+  .item {
+    padding: 0;
+    border: 0;
+  }
+  .icon {
+    width: 40px;
+    margin-top: 5px;
+    ${mq[1]} {
+      width: 40px;
+      margin-top: 0;
+    }
+  }
+  .title {
+    font-size: 20px;
+    line-height: 36px;
+    font-weight: 700;
+    color: var(--ifm-font-base-color);
+    ${mq[1]} {
+      font-size: 23px;
+      line-height: 26px;
+    }
+  }
+  .description {
+    font-size: 14px;
+    line-height: 20px;
+    color: #5f5f5f;
+    margin-top: -8px;
+    margin-bottom: 23px;
+    ${mq[1]} {
+      font-size: 17px;
+      line-height: 22px;
+      color: #484848;
+      margin-bottom: 35px;
+      margin-top: 0;
     }
   }
 `;
 
-const StyledGetInvolved = styled('div')`
-  margin-bottom: 25px;
+const StyledCalendarIframe = styled('iframe')`
+  display: block;
+  margin: 20px auto 30px;
+  max-width: 800px;
+  width: 100%;
+  height: 600px;
+  border: 0;
+  ${mq[1]} {
+    width: calc(100% - 40px);
+  }
+`;
+
+const StyledNewsletterIframe = styled('iframe')`
+  display: block;
+  max-width: 1080px;
+  width: calc(100% - 40px);
+  height: 285px;
+  margin: 30px auto 20px;
+  border: 0;
+  @media (max-width: 1200px) {
+    height: 380px;
+  }
+  @media (max-width: 679px) {
+    height: 680px;
+    margin-top: 15px;
+  }
 `;
 
 const Community = () => {
   return (
     <Layout
       title="Community"
-      description="Community website for Apache Superset, a data visualization and data exploration platform"
+      description="Community website for Apache Superset™, a data visualization and data exploration platform"
     >
-      <StyledMain>
-        <section>
-          <h1 className="title">Community</h1>
-          Get involved in our welcoming, fast growing community!
-        </section>
-        <section className="joinCommunity">
-          <StyledGetInvolved>
-            <h2>Get involved!</h2>
-            <List
-              size="small"
-              bordered
-              dataSource={links}
-              renderItem={([href, link, post]) => (
-                <List.Item>
-                  <a href={href}>{link}</a>
-                  {' '}
-                  -
-                  {' '}
-                  {post}
-                </List.Item>
-              )}
-            />
-          </StyledGetInvolved>
-        </section>
-      </StyledMain>
+      <main>
+        <BlurredSection>
+          <SectionHeader
+            level="h1"
+            title="Community"
+            subtitle="Get involved in our welcoming, fast growing community!"
+          />
+        </BlurredSection>
+        <StyledJoinCommunity>
+          <List
+            className="list"
+            itemLayout="horizontal"
+            dataSource={communityLinks}
+            renderItem={({ url, title, description, image }) => (
+              <List.Item className="item">
+                <List.Item.Meta
+                  avatar={
+                    <a className="title" href={url} target="_blank">
```

Comment:
It might be good to add link titles for accessibility. I haven't scanned through, but there may be many other places we could improve accessibility with aria labels and such.

Reply:
Yeah, I just added some aria labels where it made sense.